ports/py-qscintilla/vcpkg.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "py-qscintilla",
```

Comment:
intended to not depend on python3 ?

Reply:
indirectly via py-pyqt6, is this not ideal?

Reply:
Depends if another script port is able to load the port config from python3. But the python dependency should probably always be explicit.